### PR TITLE
Successfully switch to new discord logo. Hopefully.

### PR DESCRIPTION
**Pull request by Regular-Baf**
I'm not sure if this will actually work but it bothered me so I gave it a go.

### CODE REVIEW

**Comment by d-513** [comment]
nooo ):

File: launcher/resources/multimc/scalable/discord.svg [hunk 1]
```diff
@@ -1,108 +1,10 @@
...
+</g>
+<defs>
+<clipPath id="clip0">
+<rect width="71" height="55" fill="white"/>
+</clipPath>
+</defs>
 </svg>
```
Comment (Scrumplex) [inline launcher/resources/multimc/scalable/discord.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg fill="none" version="1.1" viewBox="0 0 71 71" xmlns="http://www.w3.org/2000/svg">
 <g transform="translate(0 8.0294)" clip-path="url(#clip0)">
  <path d="m60.104 4.8978c-4.5253-2.0764-9.378-3.6062-14.452-4.4824-0.0924-0.01691-0.1847 0.025349-0.2323 0.10987-0.6241 1.11-1.3154 2.5581-1.7995 3.6963-5.4572-0.817-10.886-0.817-16.232 0-0.4842-1.1635-1.2006-2.5863-1.8275-3.6963-0.0476-0.0817-0.1399-0.12396-0.2323-0.10987-5.071 0.87338-9.9237 2.4032-14.452 4.4824-0.0392 0.0169-0.0728 0.0451-0.0951 0.0817-9.2046 13.751-11.726 27.165-10.489 40.412 0.005597 0.0648 0.041978 0.1268 0.092353 0.1662 6.0729 4.4598 11.956 7.1673 17.729 8.9619 0.0924 0.0282 0.1903-0.0056 0.2491-0.0817 1.3657-1.865 2.5831-3.8315 3.6269-5.8995 0.0616-0.1211 0.0028-0.2648-0.1231-0.3127-1.931-0.7325-3.7697-1.6256-5.5384-2.6398-0.1399-0.0817-0.1511-0.2818-0.0224-0.3776 0.3722-0.2789 0.7445-0.5691 1.0999-0.8621 0.0643-0.0535 0.1539-0.0648 0.2295-0.031 11.62 5.3051 24.199 5.3051 35.682 0 0.0756-0.0366 0.1652-0.0253 0.2323 0.0282 0.3555 0.293 0.7277 0.586 1.1027 0.8649 0.1287 0.0958 0.1203 0.2959-0.0196 0.3776-1.7687 1.0339-3.6074 1.9073-5.5412 2.637-0.1259 0.0479-0.1819 0.1944-0.1203 0.3155 1.0662 2.0651 2.2836 4.0316 3.6241 5.8967 0.056 0.0789 0.1567 0.1127 0.2491 0.0845 5.8014-1.7946 11.684-4.5021 17.757-8.9619 0.0532-0.0394 0.0868-0.0986 0.0924-0.1634 1.4804-15.315-2.4796-28.618-10.498-40.412-0.0196-0.0394-0.0531-0.0676-0.0923-0.0845zm-36.379 32.428c-3.4983 0-6.3808-3.2117-6.3808-7.156s2.8266-7.156 6.3808-7.156c3.5821 0 6.4367 3.2399 6.3807 7.156 0 3.9443-2.8266 7.156-6.3807 7.156zm23.592 0c-3.4982 0-6.3807-3.2117-6.3807-7.156s2.8265-7.156 6.3807-7.156c3.5822 0 6.4367 3.2399 6.3808 7.156 0 3.9443-2.7986 7.156-6.3808 7.156z" fill="#5865f2"/>
 </g>
 <defs>
  <clipPath id="clip0">
   <rect width="71" height="55" fill="#fff"/>
  </clipPath>
 </defs>
</svg>
```

This should make the logo square. If you don't do this it gets stretched (at least on my system).

**Review by Scrumplex — CHANGES_REQUESTED** [review]

File: launcher/resources/multimc/scalable/discord.svg (hunk 1 above)
Comment (d-513) [reply launcher/resources/multimc/scalable/discord.svg]: not on my system

Comment (d-513) [reply launcher/resources/multimc/scalable/discord.svg]: can you send the whole SVG suggestion i'm struggling to test it with the werid github display

Comment (Scrumplex) [reply launcher/resources/multimc/scalable/discord.svg]: Screenshot of stretching


**My SVG**

Comment (d-513) [reply launcher/resources/multimc/scalable/discord.svg]: both are the same for me. Good that it fixes for you tho

**Review by d-513 — APPROVED** [review]

**Comment by DioEgizio** [comment]
lgtm but squash

**Review by Scrumplex — APPROVED** [review]
Thanks

**Comment by Regular-Baf** [comment]
> lgtm but squash

How does one squash? I can't find a button or anything for it

**Comment by kthchew** [comment]
> How does one squash? I can't find a button or anything for it

@Regular-Baf Squashing combines (“squashes”) all your commits into a single commit. AFAIK you can’t do it from GitHub.com. If you’re not very familiar with Git, it’s probably easiest to use GitHub Desktop: https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/squashing-commits

**Review by ZekeZDev — APPROVED** [review]

**Pull request merged**